codechain/keystore/KeystoreManager.py

```diff
@@ -0,0 +1,126 @@
+from jsondb.db import Database
+
+from ..crypto import blake160
+from ..crypto import generate_private_key
+from ..crypto import get_account_id_from_public
+from ..crypto import get_public_from_private
+from ..crypto import sign_ecdsa
+from .Errors import ErrorCode
+from .Errors import KeystoreError
+from .KeyType import get_table_name
+from .KeyType import KeyType
+from .StorageJson import decode
+from .StorageJson import encode
+
+
+class keystoreManager:
+    def __init__(self, key_type: KeyType, db_path: str):
+        self.db = Database(db_path)
+        self.key_type = key_type
+
+        data = {"meta": "", "platform": [], "asset": [], "hdwseed": []}
+        self.db.data(dictionary=data)
+
+    def get_keys(self) -> list:
+        rows = self.db[self.key_type.value]
```

Comment:
Why it works conditionally based on `self.key_type`?

Reply:
Please check the above comment.